internal/tiltfile/tiltfile.go

```diff
@@ -17,12 +17,16 @@ type Tiltfile struct {
 }
 
 func makeSkylarkDockerImage(thread *skylark.Thread, fn *skylark.Builtin, args skylark.Tuple, kwargs []skylark.Tuple) (skylark.Value, error) {
-	var dockerfileName, dockerfileTag skylark.String
-	err := skylark.UnpackArgs(fn.Name(), args, kwargs, "docker_file_name", &dockerfileName, "docker_file_tag", &dockerfileTag)
+	var dockerfileName, entrypoint, dockerfileTag skylark.String
```

Comment:
out of curiosity, why are we unpacking these as skylark.Strings instead of as plain strings? it's not like we ever need them as skylark.Strings again, and it seems like we might as well let UnpackArgs do the unpacking work for us

Reply:
pretty much just ignorance - at the time I wrote this, I was unaware UnpackArgs could handle plain strings, and I thought we'd need them in the data structure as strings so that we could do other things like `print(docker_image)` from the Tiltfile. It turns out these were both unfounded.